Sources/KlaviyoSwift/Models/Event.swift

```diff
@@ -10,6 +10,10 @@ import Foundation
 import KlaviyoCore
 
 public struct Event: Equatable {
+    public enum IAFProfileEvent: Equatable {
```

Comment:
Curious about the changes in the public `Event` object? Can't we just use the `properties` dictionary?